controllers/metallb_controller_test.go

```diff
@@ -115,11 +114,11 @@ var _ = Describe("MetalLB Controller", func() {
 })
 
 func cleanTestNamespace() error {
-	err := k8sClient.DeleteAllOf(context.Background(), &v1alpha1.AddressPool{}, client.InNamespace(MetalLBTestNameSpace))
```

Comment:
The addresspool_controller_test actually still uses v1alpha so we shouldn't delete it here (without changing the unit tests or something):
https://github.com/metallb/metallb-operator/blob/main/controllers/addresspool_controller_test.go#L27

Reply:
That must change. My idea was to have all beta but that single e2e tests proving that alpha still works.
Changing it.